module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -837,6 +858,23 @@ public function getMyTransactions($patron, $params = [])
             }
             $transactions[] = $transaction;
         }
+        if ($this->config['InnReach']['enabled'] ?? false) {
+            $n = 0;
+            foreach ($transactions as $transaction) {
```

Comment:
Will this work the same if you do `foreach ($transactions as $n => $transaction) {` instead of separately managing the $n counter? It looks like that might be a worthwhile simplification.